### PR TITLE
Update qcom-ipq4019-cm520-79f.dts

### PR DESCRIPTION
**Pull request by eugene91cn**
Signed-off-by: eugene91cn <113156209+eugene91cn@users.noreply.github.com>

Thanks for your contribution to OpenWrt!

To help keep the codebase consistent and readable,
and to help people review your contribution,
we ask you to follow the rules you find in the wiki at this link
https://openwrt.org/submitting-patches

Please remove this message before posting the pull request.

### CODE REVIEW

**Comment by robimarko** [comment]
PR and commit have a ton of issues.
So start by adding:

- A proper PR title
- Proper commit title and description
- SoB with a full name and real email

Make sure to explain what are you doing and why, cause I already see stuff that is not logical in the commit

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dts [hunk 1]
```diff
@@ -197,13 +201,11 @@
 };
 
 &gmac0 {
-	nvmem-cells = <&macaddr_art_1006>;
-	nvmem-cell-names = "mac-address";
+	mtd-mac-address = <&art 0x1006>;
```
Comment (robimarko) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dts:204]: NACK, unless there is a really good reason, the same applies for NVMEM getting dropped in other nodes

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dts [hunk 2]
```diff
@@ -218,95 +220,19 @@
 			#size-cells = <1>;
 
 			partition@0 {
-				label = "SBL1";
```
Comment (robimarko) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dts:221]: NACK, what is the reason for dropping individual partitions and just blobing them up as "Bootloader" which is incorrect

File: target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dts [hunk 3]
```diff
@@ -218,95 +220,19 @@
...
-					reg = <0x5000 0x2f20>;
-				};
-
-				macaddr_art_5006: macaddr@5006 {
-					reg = <0x5006 0x6>;
-				};
 			};
 
 			partition@b80000 {
-				label = "ubi";
```
Comment (robimarko) [inline target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dts:309]: NACK, unless the bootloader is passing `ubi.mtd = "rootfs"`

**Comment by mans0n** [comment]
For me it seems to be a retry of #5041, but without any improvement in formal perspective...

**Pull request closed without merging**